easyvista/icon_easyvista/util/api.py

```diff
@@ -0,0 +1,60 @@
+from insightconnect_plugin_runtime.exceptions import PluginException
+from json import JSONDecodeError
+import requests
+from requests.auth import HTTPBasicAuth
+
+
+class EasyVistaApi:
+    def __init__(self, client_login: dict, account: int, hostname: str):
+        self.base_url = f"{hostname}/api/v1/{account}/"
+        self.username = client_login.get("username")
+        self.password = client_login.get("password")
+
+    def _call_api(self, method: str, endpoint: str, json: dict = None):
+
+        response = requests.request(
+            url=self.base_url + endpoint,
```

Comment:
You don't need to fix this, but next time you might want to use

```urllib.parse.urljoin(self.base_url, endpoint)```

Then you don't have to be careful with the trailing or leading `/ `